backend/amd64/proc.ml

```diff
@@ -104,6 +104,12 @@ let register_class r =
   | Val | Int | Addr -> 0
   | Float -> 1
 
+let register_class_tag c =
+  match c with
+  | 0 -> "i"
+  | 1 -> "f"
+  | c -> Printf.sprintf "Unknown-Reg-Class(%d)" c
```

Comment:
It would be better to have Misc.fatal_error if `c is >= Proc.num_register_classes`

Reply:
Done